src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/TestUtil.java

```diff
@@ -40,6 +44,16 @@
 
 public final class TestUtil {
 
+    /**
+     * The stack size used in {@link #executeWithLimitedStackSize(Executable)}.
+     * This value should be as small as possible.
+     *
+     * @see <a href="https://www.baeldung.com/jvm-configure-stack-sizes">
+     *      Configuring Stack Sizes in the JVM</a>
+     * @noinspection MultiplyOrDivideByPowerOfTwo
```

Comment:
IDEA suggests to replace `144 * 1024` with `144 << 10`, but in such case is makes code less readable.

Reply:
I suggest to change it to number with good description in javadoc instead of mystic calculations, thus will eliminate all warmings

Reply:
done